src/js/msp/MSPHelper.js

```diff
@@ -536,6 +536,18 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 SENSOR_ALIGNMENT.align_gyro = data.readU8();
                 SENSOR_ALIGNMENT.align_acc = data.readU8();
                 SENSOR_ALIGNMENT.align_mag = data.readU8();
+
+                if (semver.gte(CONFIG.apiVersion, '1.41.0')) {
+
+                    var gyro_detection_flags = data.readU8();
+                    SENSOR_ALIGNMENT.detected_gyro_1 = (gyro_detection_flags & (1 << 0)) != 0;
```

Comment:
Instead of splitting this here, I think it is better to store it here into a single variable, and put the splitting code into `configuration.js` where it is used - this will mean only one place needs to be updated if we ever change the contents of this byte (like add support for 3 or more gyros).

Reply:
I did it here precisely to have only one place where we split the value. If I do it in the place where it is used, I will need to do it the same if used in other tabs.
But I will change it, no problem 😉